include/ceed/types.h

```diff
@@ -13,6 +13,22 @@
 #include <stddef.h>
 #include <stdint.h>
 
+/**
+ This macro defines compiler attributes to the CEED_QFUNCTION to force inlining
+ for called functions. The `inline` declaration does not necessarily enforce a
+ compiler to inline a function. This can be deterimental to performance, so
+ here we force inlining to occur.
+**/
```

Comment:
I'm leaving of a `@ingroup` specification here as this macro shouldn't really be used by front-end or back-end developers. Let me know if there's a place for this though.

Reply:
Does the Python failure stay if you put in an ingroup specification?

Reply:
Yep.

Reply:
That error means that the Python is choking while parsing this header as part of setting up the Python interface